ui/components/app/assets/nfts/nft-details/nft-detail-information-row.tsx

```diff
@@ -57,7 +84,37 @@ const NftDetailInformationRow: React.FC<NftDetailInformationRowProps> = ({
           color={valueColor || TextColor.textAlternative}
           variant={TextVariant.bodyMdMedium}
         >
-          {value}
+          {withPopover && fullValue ? (
+            <>
+              <Box
+                onMouseEnter={handleMouseEnter}
+                onMouseLeave={handleMouseLeave}
+                ref={setBoxRef}
+              >
+                {value}
+              </Box>
+              <Popover
```

Comment:
Any reason the `<Tooltip>` component was not used?

Reply:
The designs [here](https://www.figma.com/design/TfVzSMJA8KwpWX8TTWQ2iO/Asset-list-and-details?node-id=4313-14410&m=dev) use popover and not tooltip